<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>实验室信息管理系统</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="acc_Layout">
     <item>
      <widget class="QLabel" name="acc_label">
       <property name="text">
        <string>账号</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="acc_Edit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="pwd_Layout">
     <item>
      <widget class="QLabel" name="pwd_label">
       <property name="text">
        <string>密码</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pwd_Edit">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="identity_Layout">
     <item>
      <widget class="QLabel" name="identity_label">
       <property name="text">
        <string>身份</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="stu_radioBtn">
       <property name="text">
        <string>学生</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="teacher_radioBtn">
       <property name="text">
        <string>教师</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="admin_radioBtn">
       <property name="text">
        <string>管理员</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="loginButton">
     <property name="text">
      <string>登录</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
